src/databases/Curve3D/avtCurve3DOptions.C

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Lawrence Livermore National Security, LLC and other VisIt
+// Project developers.  See the top-level LICENSE file for dates and other
+// details.  No copyright assignment is required to contribute to VisIt.
+
+// ************************************************************************* //
+//                          avtCurve3DOptions.C                              //
+// ************************************************************************* //
+
+#include <avtCurve3DOptions.h>
+
+#include <DBOptionsAttributes.h>
+
+#include <string>
+#include <vector>
+
+using std::string;
+using std::vector;
+
+// ****************************************************************************
+//  Function: GetCurve3DReadOptions
+//
+//  Purpose:
+//      Creates the options for Curve3D readers. (currently none)
+//
+//  Programmer: Kathleen Biagas
+//  Creation:   August 31, 2018
+//
+// ****************************************************************************
+
+DBOptionsAttributes *
+GetCurve3DReadOptions(void)
+{
+    DBOptionsAttributes *rv = new DBOptionsAttributes;
+    return rv;
+}
+
+
+// ****************************************************************************
+//  Function: GetCurve3DWriteOptions
+//
+//  Purpose:
+//      Creates the options for Curve3D writers.
+//
+//  Programmer: Kathleen Biagas 
+//  Creation:   August 31, 2018
+//
+//  Modifications:
+//
+// ****************************************************************************
+
+DBOptionsAttributes *
+GetCurve3DWriteOptions(void)
+{
+    DBOptionsAttributes *rv = new DBOptionsAttributes;
+    vector<string> commentStyle;
+    commentStyle.push_back("# (Ultra)");
```

Comment:
@biagas does Ultra support 3D curves?

Reply:
I don't recall, it's been a long while since I played around with Ultra.

Reply:
Ok, we'll leave it in in lieu of any information to the contrary.